emptytheme/block-templates/index.html

```diff
@@ -0,0 +1,9 @@
+<!-- wp:template-part {"slug":"header","theme":"emptytheme","tagName":"header"} /-->
```

Comment:
I think we should not force a theme to specify the theme on each place the theme references a template part. The core should automatically assume that the template part being referenced is from that theme.

Reply:
Yeah, definitely. This seems like an unnecessary attribute in this case.